editor/src/components/canvas/canvas-wrapper-component.tsx

```diff
@@ -131,6 +92,21 @@ const ErrorOverlayComponent = betterReactMemo(
       return getAllCodeEditorErrors(store.editor, true, true)
     }, 'ErrorOverlayComponent fatalCodeEditorErrors')
 
+    const runtimeErrors = useReadOnlyRuntimeErrors()
+
+    const overlayErrors = React.useMemo(() => {
+      return runtimeErrors.map((runtimeError) => {
+        const stackFrames =
+          runtimeError.error.stackFrames != null ? runtimeError.error.stackFrames : []
```

Comment:
`defaultIfNull`?

Reply:
this is just moved code, I thought it's best if I keep it as-is for easier brain processing of PR